### PR TITLE
Show User Login Tracking Info in Admin

### PR DESCRIPTION
**Pull request by tubbo**
Display the `last_login_at` and `last_login_attempted_at` properties of a User on the attributes card in admin.

### CODE REVIEW

File: admin/app/views/workarea/admin/users/_cards.html.haml [hunk 1]
```diff
@@ -35,6 +35,12 @@
                 = link_to model.created_by.name, user_path(model.created_by)
               - else
                 = t('workarea.admin.users.cards.not_applicable')
+            %li
+              %strong= t('workarea.admin.fields.last_logged_in_at')
+              = model.last_login
```
Comment (tubbo) [inline admin/app/views/workarea/admin/users/_cards.html.haml:40]: This method already existed, but wasn't in use.

File: admin/config/locales/en.yml [hunk 2]
```diff
@@ -1718,6 +1718,8 @@ en:
         last_name: Last Name
         last_order: Last Order
         last_order_at: Last Order At
+        last_logged_in_at: Last Logged In At
```
Comment (tubbo) [inline admin/config/locales/en.yml:1721]: dunno if this should really have the "At" on the end, might read a little weird..

File: admin/app/views/workarea/admin/users/_cards.html.haml (hunk 1 above)
Comment (mttdffy) [reply admin/app/views/workarea/admin/users/_cards.html.haml:40]: That method is using `time_ago_in_words` in a view model, which I think that method is not defined in and probably throwing an error to cause the test in the build to fail.

File: admin/config/locales/en.yml (hunk 2 above)
Comment (mttdffy) [reply admin/config/locales/en.yml:1721]: I'd say drop the `at`

File: admin/app/views/workarea/admin/users/_cards.html.haml (hunk 1 above)
Comment (mttdffy) [reply admin/app/views/workarea/admin/users/_cards.html.haml:40]: err, ok it includes the date helper.. so maybe its not that. Build is failing though

Comment (tubbo) [reply admin/app/views/workarea/admin/users/_cards.html.haml:40]: should be resolved once https://github.com/workarea-commerce/workarea/pull/490 is merged.

Comment (bencrouse) [reply admin/app/views/workarea/admin/users/_cards.html.haml:40]: Yeah, that's the same failure that I was seeing locally.

**Comment by tubbo** [comment]
I fixed the test you were talking about @bencrouse but I think this final failure has to do with impersonation, so i think a mergedown will greenify the build.

File: admin/app/views/workarea/admin/users/edit.html.haml [hunk 3]
```diff
@@ -80,12 +80,24 @@
...
+                = label_tag 'user_last_login_attempt_at', t('workarea.admin.fields.last_login_attempt_at'), class: 'property__name'
+                = local_time(@user.last_login_attempt_at)
+
           - if @user.created_by.present?
             .grid__cell.grid__cell--50.grid__cell--25-at-medium
               .property
                 = label_tag 'user_created_by', t('workarea.admin.fields.created_by'), class: 'property__name'
                 = link_to @user.created_by.name, user_path(@user.created_by)
 
+
```
Comment (bencrouse) [inline admin/app/views/workarea/admin/users/edit.html.haml:100]:
```suggestion
```

**Pull request merged**